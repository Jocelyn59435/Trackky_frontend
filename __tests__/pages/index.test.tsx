import React from 'react';
import renderer from 'react-test-renderer';
import Index from '../../pages/index';

it('renders homepage correctly', () => {
  const tree = renderer.create(<Index />).toJSON();
  expect(tree).toMatchSnapshot();
});
