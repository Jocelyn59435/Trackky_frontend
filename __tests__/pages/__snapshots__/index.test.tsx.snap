// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage correctly 1`] = `
Array [
  <div
    className="bg-indigo py-3 flex justify-between items-center"
  >
    <div
      className="p-2 cursor-pointer"
    >
      <a
        className="cursor-pointer"
        href="/"
      >
        <div
          style={
            Object {
              "boxSizing": "border-box",
              "display": "inline-block",
              "height": 50,
              "overflow": "hidden",
              "position": "relative",
              "width": 180,
            }
          }
        >
          <img
            data-nimg="fixed"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              Object {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              data-nimg="fixed"
              decoding="async"
              loading="lazy"
              src="/_next/image?url=%2Ftrackky_header.png&w=384&q=75"
              srcSet="/_next/image?url=%2Ftrackky_header.png&w=256&q=75 1x, /_next/image?url=%2Ftrackky_header.png&w=384&q=75 2x"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </div>
      </a>
    </div>
    <div
      onClick={[Function]}
      onMouseEnter={[Function]}
      style={Object {}}
    >
      <button
        className="lg:mr-20 md:mr-10 sm: mr-5 w-28 h-12 font-bold bg-grey text-indigo text-center rounded-lg text-xl"
      >
        Log In
      </button>
    </div>
  </div>,
  <div
    className="m-auto w-5/6 justify-around flex lg:flex-row md:flex-row sm:flex-col"
  >
    <div
      className="space-y-2 p-20"
    >
      <p
        className="text-blue text-4xl font-semibold font-mono tracking-wider"
      >
        Enter the Product Link
      </p>
      <p
        className="text-blue text-4xl font-semibold font-mono tracking-wider"
      >
        Get Your Price Drop Email Alert
      </p>
      <p
        className="text-middlegrey text-2xl font-bold font-mono tracking-wide"
      >
        Trackky is here to help you save the money,
      </p>
      <p
        className="text-middlegrey text-2xl font-bold font-mono tracking-wide"
      >
        enjoy goodies at the right time.
      </p>
      <p
        className="font-mono font-semibold tracking-wider"
      >
        Current Range:
      </p>
      <p>
        Chemist Warehouse
      </p>
    </div>
    <div
      className="m-auto"
    >
      <div
        style={
          Object {
            "boxSizing": "border-box",
            "display": "inline-block",
            "margin": 0,
            "maxWidth": "100%",
            "overflow": "hidden",
            "position": "relative",
          }
        }
      >
        <div
          style={
            Object {
              "boxSizing": "border-box",
              "display": "block",
              "maxWidth": "100%",
            }
          }
        >
          <img
            alt=""
            aria-hidden={true}
            src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzUwIiBoZWlnaHQ9IjM1MCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
            style={
              Object {
                "border": "none",
                "display": "block",
                "margin": 0,
                "maxWidth": "100%",
                "padding": 0,
              }
            }
          />
        </div>
        <img
          data-nimg="intrinsic"
          decoding="async"
          src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
          style={
            Object {
              "border": "none",
              "bottom": 0,
              "boxSizing": "border-box",
              "display": "block",
              "height": 0,
              "left": 0,
              "margin": "auto",
              "maxHeight": "100%",
              "maxWidth": "100%",
              "minHeight": "100%",
              "minWidth": "100%",
              "objectFit": undefined,
              "objectPosition": undefined,
              "padding": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": 0,
            }
          }
        />
        <noscript>
          <img
            data-nimg="intrinsic"
            decoding="async"
            loading="lazy"
            src="/_next/image?url=%2FanalyticsIcon.svg&w=750&q=75"
            srcSet="/_next/image?url=%2FanalyticsIcon.svg&w=384&q=75 1x, /_next/image?url=%2FanalyticsIcon.svg&w=750&q=75 2x"
            style={
              Object {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
        </noscript>
      </div>
    </div>
  </div>,
]
`;
